lsp-mode.el

```diff
@@ -38,10 +38,11 @@
   :lighter (:eval (lsp-mode-line))
   :group 'lsp-mode)
 
-(defun lsp--make-stdio-connection (name command command-fn stderr)
+(defun lsp--make-stdio-connection (name command command-fn get-root-fn stderr)
   (lambda (filter sentinel)
     (let* ((command (if command-fn (funcall command-fn) command))
-           (final-command (if (consp command) command (list command))))
+           (final-command (if (consp command) command (list command)))
```

Comment:
The root is decided in `lsp--start`. This will not work as `get-root-fn` may be nil (I have PRs to change some clients, https://github.com/emacs-lsp/lsp-python/pull/26) and I think it is recommended to use `nil`.